docs/start/stack.rst

```diff
@@ -76,50 +53,21 @@ Preparation of client-side resources involves:
 
-*TODO: is this accurate?*
+Automated testing
+-----------------
 
+We use a number of mechanisms to help encourage code quality and consistency. Most of these are run automatically on Github pull requests, and developers should run them locally too.
 
-Installers and Packages
-~~~~~~~~~~~~~~~~~~~~~~~
-
-*TODO: introduce stack (sdist, PyPi, Debian, Windows, etc)*
-
-see :ref:`build_pipeline`
-
-
-Continuous Integration
-~~~~~~~~~~~~~~~~~~~~~~
-
-*TODO: introduce stack (GitHub, CodeCov, Travis, buildkite, commit hooks)*
-
-
-Tests and Linting
-~~~~~~~~~~~~~~~~~
-
-We use a number of mechanisms to help encourage code quality and consistency. These checks enforce a subset of our :ref:`conventions`.
-
-- `pre-commit <http://pre-commit.com/>`_ is run locally on ``git commit`` and enforces some Python conventions
+- `pre-commit <http://pre-commit.com/>`_ is run locally on ``git commit`` and enforces a variety of code conventions
 - We use `EditorConfig <http://editorconfig.org/>`_ to help developers set their editor preferences
-- `flake8 <https://flake8.readthedocs.io/en/latest/>`_ is also used to enforce Python conventions
 - `tox <https://tox.readthedocs.io/en/latest/>`_ is used to run our test suites under a range of Python and Node environment versions
 - ``sphinx-build -b linkcheck`` checks the validity of documentation links
 - `pytest <http://pytest.org/latest/>`_ runs our Python unit tests. We also leverage the `Django test framework <https://docs.djangoproject.com/en/1.9/topics/testing/>`_.
 - In addition to building client assets, `webpack <https://webpack.github.io/>`_ runs linters on client-side code: `ESLint <http://eslint.org/>`_ for ES6 JavaScript, `Stylint <https://rosspatton.github.io/stylint/>`_ for Stylus, and `HTMLHint <http://htmlhint.com/>`_ for HTML and Vue.js components.
-- Client-side code is tested using a stack of tools including `Karma <https://karma-runner.github.io/0.13/index.html>`_, `Mocha <https://mochajs.org/>`_, `PhantomJS <http://phantomjs.org/>`_, `Sinon <http://sinonjs.org/>`_, and `rewire <https://github.com/jhnns/rewire>`_. *TODO: Explain what each of these do*
-- `codecov <https://codecov.io/>`_ reports on the test coverage for Python and Node.js code. *TODO - also client-side?*
-
-
-Helper Scripts
----------------
-
-*TODO: introduce stack (kolibri command, setup.py, makefiles, yarn commands, sphinx auto-build, etc)*
-
-
+- Client-side code is tested using `Jest <https://facebook.github.io/jest/>`_ and `Sinon <http://sinonjs.org/>`_
```

Comment:
Note, once #3797 is merged, can remove the reference to Sinon.

Reply:
thanks - i'll just remove it now in anticipation

Reply:
It is now also merged!